routes/index.js

```diff
@@ -1,5 +1,11 @@
+"use strict"
 var express = require('express');
 var router = express.Router();
+var pgp = require('pg-promise')();
+var pq = require('pg-promise').ParameterizedQuery;
+
+const connectionstring = `postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.POSTGRES_HOST}:${process.env.POSTGRES_PORT}/${process.env.POSTGRES_DB}`;
+const db = pgp(connectionstring);
```

Comment:
Please don't make a connection at the root scope of the module. If this connection is slow or fails the module won't load. The guide for postgres in nodeJS recommends that the connection should be made in the response handler.